packages/app/client/src/App.js

```diff
@@ -40,73 +40,100 @@ import {
 } from './ui'
 
 // pages
-import { Register, Login, ForgotPassword, ResetPassword, Learn } from './ui/pages'
+import {
+  Register,
+  Login,
+  ForgotPassword,
+  ResetPassword,
+  Learn
+} from './ui/pages'
 import projectsRoutes from './routes/projects'
 
 require('./utils/configureRequests')
 
-getUserInformation()
-
 const App = () => {
+  const [userInfoSet, setUserInfoSet] = useState(false)
+  useEffect(async () => {
+    await getUserInformation()
+    setUserInfoSet(true)
+  }, [])
```

Comment:
hey man, could you change to `const fetchUser = async () => { await getUserInformation()}` and then call it to avoid strange behaviour  because when we use useEffect like in your implementation we receive a warning like this 
![useEffect_error](https://user-images.githubusercontent.com/12929857/116691410-aefcad80-a9b2-11eb-9415-56774d49d37a.png)

Reply:
It is a suggestion from Dan Abramov -> https://github.com/facebook/react/issues/14326